static/src/stylesheets/module/identity/_manage-account-tab.scss

```diff
@@ -269,6 +269,30 @@
             }
         }
     }
+
+    /* Cancellation Contribution section
+    ===================================================================== */
+    .contribution-cancellation__menu {
+        border-top: solid 1px black;
+
+        h3 {
+            margin-top: 28px;
+            margin-bottom: 26px;
+        }
+
+        contribution-cancellation__menu__actions {
+            margin-top: 56px;
+        }
+    }
+
+    .manage-account__button.is-disable {
+        cursor: not-allowed;
+        background-color: #bdbdbd;
+    }
+
+    .manage-account__button.is-disable:hover {
+        background-color: #bdbdbd;
+    }
```

Comment:
Can we replace the above by adding `.:not(.is-disable)` to [this line](https://github.com/guardian/frontend/blob/8272df00a132f0919e9344fb458bbaa26facd375/static/src/stylesheets/module/identity/_manage-account-tab.scss#L449)?

Reply:
Also, slightly pedantic, can we rename `is-disable` to `is-disabled`?